VERSION

```diff
@@ -0,0 +1 @@
+1.22.0-12.0
```

Comment:
Is this the same version as what the plugin will be in the registry? The idea is that we want to keep the version field here and the plugin version the same.

Reply:
it's the sdk version actually

Reply:
@ericwill why the framework image needs to be the same than the extension version ?

if we have two VS Code extensions using flutter, they could reuse the same image